redi-connect-front/src/i18n.tsx

```diff
@@ -0,0 +1,27 @@
+import i18n from 'i18next'
```

Comment:
Also, let's move this file into the `services` folder

Reply:
done